app/models/concerns/parameterizable.rb

```diff
@@ -12,7 +12,7 @@ def to_param
       end
 
       def self.from_param(id)
-        self.find(id.to_i)
+        self.find_by_id(id.to_i)
```

Comment:
`find_by_id` doesn't fail with an exception ActiveRecord::RecordNotFound`, and allows us to handle gracefully `from_param` calls, usually from FindCommon.